pkg/services/auth/session_cleanup.go

```diff
@@ -0,0 +1,38 @@
+package auth
```

Comment:
This file is named session_cleanup - does that imply that we should rename the UserAuthTokenService/user_auth_token to UserSessionService/user_session?

Reply:
good point. lets circle back to this post merge.